invenio_records_rest/views.py

```diff
@@ -74,6 +76,11 @@ def validation_error(error):
         """Catch validation errors."""
         return RESTValidationError().get_response()
 
+    @blueprint.errorhandler(RequestError)
+    def elasticsearch_badrequest_error(error):
+        """Catche problems in ElasticSearch queries."""
+        return BadRequest(description=error.error).get_response()
```

Comment:
The error message has to be rewritten to something meaningful for an end-user as the message is display upfront in the UI. Please also see ``invenio_rest.errors``